wiki/Internet_Relay_Chat/en.md

```diff
@@ -35,47 +32,32 @@ Once you have a client, you need to set up the server settings.
-| [KVIrc](https://www.kvirc.net/) | Visit [this thread](http://www.kvirc.ru/forum/?topic=609.0) in the official KVIrc forums. |
-| [mIRC](https://www.mirc.com/) | Go to mIRC Options (Tools - Options / Alt + O), under IRC tree, click "Events..." button and change Joins and Parts to "Hide" |
-| [Quassel IRC](https://quassel-irc.org/) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit. |
-| [XChat](http://xchat.org/) | Type `/set irc_conf_mode 1` (or [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels). |
+| [HexChat](https://hexchat.github.io/ "GitHub") | Check "Hide join and part messages" under Settings - Preferences -> Chatting -> General (Advanced pre-2.9.6)  |
+| [ircII](http://www.eterna.com.au/ircii/ "ircII") | Type `IGNORE * CRAP` |
+| [Irssi](https://irssi.org "Irssi") | Type `/ignore -channels #somechannel * JOINS PARTS QUITS` |
+| [Weechat](https://weechat.org/ "Weechat") | Type `/filter add joinquit * irc_join,irc_part,irc_quit *`. |
+| [KVIrc](https://www.kvirc.net/ "KVIrc") | (Refer to [the KVIrc wiki page](https://github.com/kvirc/KVIrc/wiki/FAQ#how-do-i-suppress-join-part-and-quit-messages "GitHub")) |
+| [mIRC](https://www.mirc.com/ "mIRC") | Go to mIRC Options (Tools - Options / Alt + O), under IRC tree, click "Events..." button and change Joins and Parts to "Hide" |
+| [Quassel IRC](https://quassel-irc.org/ "Quassel IRC") | Right-click inside the chat and choose Joins/Parts/Quits in the "Hide Events" menu. |
+| [XChat](http://xchat.org/ "XChat") | Type `/set irc_conf_mode 1` |
 
-If your client is not listed here refer to its documentation, most clients have a way to do this.
-
-## Frequently asked questions (FAQ)
+## FAQ
 
 ### I'm receiving the error "Bad Authentication Token"
 
-1. Make sure you are using the password at the [IRC Authentication](https://osu.ppy.sh/p/irc) page.
+If you're recieving this error, you should first try the following:
+
+1. Make sure you are using the correct password at the [IRC Authentication page](https://osu.ppy.sh/p/irc).
 2. If your username has spaces, replace it to underscores (e.g. **This Username** to **This_Username**)
 
 ### Can I use another username?
 
 No. You can only use your osu! username.
 
-### What is this voice status that I have? I also see some people having it
-
-Users with *voice status* are also connected using an IRC client, except for chat moderators who always have *operator (+o)* status regardless of what client they use.
+### What is this "voice status" that I have?
 
-Users who do not have any status are connected using the game client.
+Users with *voice status* are also connected using an IRC client (except for chat moderators who always have *operator (+o)* status, regardless of what client they use); users who do not have any status are connected using the game client.
```

Comment:
```suggestion
Users with *voice status* are also connected using an IRC client (except for chat moderators who always have *operator (+o)* status, regardless of what client they use). Users who do not have any status are connected using the game client.
```
I don't think a semicolon is needed here. just separate it with a full stop into two different sentences

Reply:
Also suggest changing 'regardless of what client' to 'regardless of which client' in this line.

Reply:
`regardless of the client` sounds better but good catch!